aptos-move/framework/src/release_bundle.rs

```diff
@@ -284,30 +285,33 @@ impl ReleasePackage {
     fn generate_next_execution_hash_blob(
```

Comment:
Can we reuse `generate_next_execution_hash_blob()` from `aptos-move/aptos-release-builder/src/utils.rs`? The code seems identical, and it is annoying that we have so much copy paste everywhere - so easy to make mistakes.

Reply:
Let me see if I can remove one of the duplicates. It might have to be the other way around due to how dependencies are set up.